managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -845,11 +845,12 @@ public ManagedCursor openCursor(String cursorName) throws InterruptedException,
     @Override
     public ManagedCursor openCursor(String cursorName, InitialPosition initialPosition)
             throws InterruptedException, ManagedLedgerException {
-        return openCursor(cursorName, initialPosition, Collections.emptyMap());
+        return openCursor(cursorName, initialPosition, Collections.emptyMap(), Collections.emptyMap());
```

Comment:
we may be better of with something like `final static Map<blah, blah> = Maps.ImmutableMap();` as a field to avoid creating empty maps adding to short-lived memory garbage.

Reply:
Same as above